geyes/src/geyes.c

```diff
@@ -144,9 +145,17 @@ timer_cb (EyesApplet *eyes_applet)
             gdk_window_get_device_position (gtk_widget_get_window (eyes_applet->eyes[i]),
                                             gdk_seat_get_pointer (seat),
                                             &x, &y, NULL);
+
+#ifdef ENABLE_IN_PROCESS
+            gdk_window_get_origin (gtk_widget_get_window (GTK_WIDGET(eyes_applet->applet)),
+                                       &applet_x, &applet_y);
```

Comment:
Indent

Reply:
If you look at the lines above, this block looks like, but is NOT inside the ` if` block that is in turn inside a `for` block, that's what fooled me. It is inside the` for` block only, fixing this now

Reply:
Correction posted above. Original bracket location hard to see so moving it

Reply:
Pushed bracket change to in-process-squashed-to-one as below:

```
    for (i = 0; i < eyes_applet->num_eyes; i++)
    {
        if (gtk_widget_get_realized (eyes_applet->eyes[i]))
        {
            gdk_window_get_device_position (gtk_widget_get_window (eyes_applet->eyes[i]),
                                            gdk_seat_get_pointer (seat),
                                            &x, &y, NULL);

#ifdef ENABLE_IN_PROCESS
            gdk_window_get_origin (gtk_widget_get_window (GTK_WIDGET(eyes_applet->applet)),
                                       &applet_x, &applet_y);
#else
            applet_x = 0;
            applet_y = 0;
#endif
            gtk_widget_translate_coordinates (eyes_applet->eyes[i],
                                              gtk_widget_get_toplevel(eyes_applet->eyes[i]),
                                              -applet_x, -applet_y, &dx, &dy);
            x -= dx;
            y -= dy;

            if ((x != eyes_applet->pointer_last_x[i]) ||
                (y != eyes_applet->pointer_last_y[i])) {

                calculate_pupil_xy (eyes_applet, x, y, &pupil_x, &pupil_y,
                                    eyes_applet->eyes[i]);
                draw_eye (eyes_applet, i, pupil_x, pupil_y);

                eyes_applet->pointer_last_x[i] = x;
                eyes_applet->pointer_last_y[i] = y;
            }
        }
    }
    return TRUE;
```

Reply:
This should be easier to read and less likely to trip up anyone working on it later